pkg/health/health.go

```diff
@@ -66,25 +67,41 @@ func (s *ServiceNodeStatus) IsCurrentVersion(version string) bool {
 
 type ConsulHealthChecker struct {
 	Health *consulapi.Health
+	Store  kp.Store
 }
 
-func NewConsulHealthChecker(consulHealth *consulapi.Health) *ConsulHealthChecker {
+func NewConsulHealthChecker(store kp.Store, consulHealth *consulapi.Health) *ConsulHealthChecker {
 	return &ConsulHealthChecker{
 		Health: consulHealth,
+		Store:  store,
 	}
 }
 
-func (s *ConsulHealthChecker) toNodeStatus(entry consulapi.ServiceEntry) *ServiceNodeStatus {
+func (s *ConsulHealthChecker) toNodeStatus(serviceID string, entry consulapi.ServiceEntry) (*ServiceNodeStatus, error) {
+
+	version := ""
+	manifest, _, err := s.Store.Pod(kp.RealityPath(entry.Node.Node, serviceID))
+	if err != nil {
```

Comment:
Wondering whether I should handle `pods.NoCurrentManifest` here specially, such as `err != nil && err != pods.NoCurrentManifest`